### PR TITLE
Changing display of add other links to dropdown at sm and md resolutions

### PR DESCRIPTION
**Pull request by rhamilto**
so that a third link can be accommodated.

Note:  this screenshot shows a third link, but that's just for mock purposes.  @jeff-phillips-18 is to add the third link.
screen shot 2017-08-17 at 9 08 38 am

### CODE REVIEW

**Comment by cshinn** [comment]
Looks great @rhamilto! "Custom Add" should be in headline case to conform to PF guidelines though

**Comment by rhamilto** [comment]
> Looks great @rhamilto! "Custom Add" should be in headline case to conform to PF guidelines though

Thanks, @cshinn.  I've fixed the casing.

File: src/components/services-view/services-view.html [hunk 1]
```diff
@@ -5,14 +5,34 @@
...
       <h1>Browse Catalog</h1>
-      <ul class="add-other" ng-if="$ctrl.onDeployImageSelected || $ctrl.onFromFileSelected">
-        <li ng-if="$ctrl.onDeployImageSelected">
-          <a href="" ng-click="$ctrl.onDeployImageSelected()">Deploy Image</a>
-        </li>
-        <li ng-if="$ctrl.onFromFileSelected">
-          <a href="" ng-click="$ctrl.onFromFileSelected()">Import <span class="import-file-types">YAML / JSON</span></a>
-        </li>
-      </ul>
+      <div ng-if="$ctrl.onDeployImageSelected || $ctrl.onFromFileSelected">
```
Comment (jeff-phillips-18) [inline src/components/services-view/services-view.html]: I know its an odd case but if there is only one item should this be a dropdown?

Comment (rhamilto) [inline src/components/services-view/services-view.html]: That would be the ideal UX, but I'm not sure how common it'll be.  Any idea, @spadgett?

Comment (spadgett) [inline src/components/services-view/services-view.html]: I think in practice you will either have all 3 (running in the console) or none. @jeff-phillips-18 OK to leave as is?

Comment (jeff-phillips-18) [inline src/components/services-view/services-view.html]: Yeah for sure

**Pull request merged**